flink-runtime/src/main/java/org/apache/flink/runtime/rpc/akka/AkkaRpcServiceUtils.java

```diff
@@ -100,11 +100,11 @@ public static RpcService createRpcService(
 			int port,
 			Configuration configuration) throws Exception {
 		final ActorSystem actorSystem = BootstrapTools.startActorSystem(configuration, hostname, port, LOG);
-		return instantiateAkkaRpcService(configuration, actorSystem);
+		return createRpcService(configuration, actorSystem);
 	}
 
 	@Nonnull
-	private static RpcService instantiateAkkaRpcService(Configuration configuration, ActorSystem actorSystem) {
+	public static RpcService createRpcService(Configuration configuration, ActorSystem actorSystem) {
```

Comment:
Introduce a method `#createRpcService(String, String, Configuration, String, BootstrapTools.ActorSystemExecutorConfiguration)` instead of make `#instantiateAkkaRpcService(Configuration, ActorSystem) ` public